zulip/integrations/bridge_with_slack/run-slack-bridge

```diff
@@ -18,23 +18,25 @@ ZULIP_MESSAGE_TEMPLATE = "**{username}**: {message}"
 SLACK_MESSAGE_TEMPLATE = "<{username}> {message}"
 
 
-def check_zulip_message_validity(
+def get_zulip_message_slack_channel(
     msg: Dict[str, Any], zulip_to_slack_map: Dict[str, Any], bot_email: str
-) -> Tuple[bool, str]:
+) -> Union[str, None]:
```

Comment:
We can use `Optional[str]` here.

Reply:
I intentionally didn't use `Optional` because this is different from the situation with `arg=None` in function arguments. It'd be better with `str | None`, but this is not supported in Python <3.10 unless there is `from __future__ import annotations`. See https://peps.python.org/pep-0604/.

Reply:
OK I will just do what is done in zulip/zulip.